src/app/shared/mangoQueries.ts

```diff
@@ -21,3 +21,12 @@ export function findAllDocuments(selector, query) {
     }
   `);
 }
+
+export function findMultiDocuments(selector, query) {
+  return JSON.parse(`
+    {
+      "selector": { "$and": [{"${selector}": "${query}"}] },
```

Comment:
so far it will only take one field and one condition, nothing more

Reply:
no it takes multiple field ..

Reply:
how will it take multiple condition here do have example tested out?